src/main/java/com/github/wovnio/wovnjava/Settings.java

```diff
@@ -14,8 +14,8 @@
     // Default configuration values
     public static final int DefaultTimeout = 1000;
     public static final String DefaultApiUrlBase  = "https://wovn.global.ssl.fastly.net";
+    public static final String DefaultWidgetVersionUrlPrefix = "https://cdn.wovn.io/widget/";
```

Comment:
Ok so wovnjava already knew about versioned snippet URLs for the purposes of removing snippets from the page. So I think we should keep this behavior, in addition to adding the snippetUrl setting.

The previous variable was more correct though, use that 
`public static final String DefaultVersionedWidgetUrlProduction = DefaultApiUrlBase + "/widget";`

Reply:
This one is just simply unused. When removing snippet URL the source comes from the `snippetUrl` defined in the setting.

Reply:
OK I guess that's fine. Basically we're going to trust that if they are using versioning, they are going to use this setting